main.js

```diff
@@ -34,40 +40,146 @@ buttonMeditate.addEventListener('click', changeColorPurple);
 buttonStudy.addEventListener('click', changeColorGreen);
 buttonExercise.addEventListener('click', changeColorOrange);
 buttonStart.addEventListener('click', showActivity);
-startTimer.addEventListener('click', starter)
+startTimer.addEventListener('click', starter);
+buttonLog.addEventListener('click', pressLog);
+buttonNewActivity.addEventListener('click', goHome);
 
 var currentActivity = '';
 var previousActivities = [];
 
+function saveActivities() {
+  currentActivity.saveToStorage()
+  previousActivities.unshift(currentActivity.id)
+
+}
+
+
+ function pressLog() {
+   startTimer.addEventListener('click', starter);
+   clockNumbers.classList.add('min')
+   clockNumbers.classList.remove('inspiration')
+   saveActivities()
+   logTheActivity();
+}
+
+function showMessage() {
+  var category = categoryButtonFinder();
+  var color = '';
+  if(category === 'Study') {
+    color = 'green';
+  } else if (category === 'Meditate') {
+    color = 'purple';
+  } else if (category === 'Exercise') {
+    color = 'orange';
+  }
+  clockNumbers.innerHTML= `<section class="message ${color}"> <h5>${messages[getRandomMessage(messages)]}</h5> </section>`;
+  clockNumbers.classList.remove('min');
+  clockNumbers.classList.add('inspiration')
+  startTimer.addEventListener('click', starter);
+}
+
+function getRandomMessage(array) {
+  return Math.floor(Math.random() * array.length);
+}
+
+function startToComplete() {
+  var startButton = document.querySelector("#startButtonText");
+  startButton.innerText = `${'COMPLETE!'}`;
```

Comment:
Does this need to be Interpolating a string? Or can we just concatenate 'COMPLETE!' into the HTML?

Reply:
Yes, I agree. I believe it could just be startButton.innerText = 'COMPLETE!'